samples/rest/count_tokens.sh

```diff
@@ -16,6 +16,11 @@ else
   B64FLAGS="-w0"
 fi
 
+echo "[START tokens_context_window]"
+# [START tokens_context_window]
+curl https://generativelanguage.googleapis.com/v1beta/models/gemini-1.5-pro:countTokens?key=$GOOGLE_API_KEY
```

Comment:
Please test things before uploading. 

Here you want to just get the model, without the ":countTokens".

Then print the input and output token limits with:

```
jq .inputTokenLimit model.json
jq .outputTokenLimit model.json
```

Reply:
Updated. This is what the output looks like now:

```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   387    0   387    0     0   3398      0 --:--:-- --:--:-- --:--:--  3424
2097152
8192
```